src/components/BirdDetails/BirdDetails.js

```diff
@@ -1,16 +1,27 @@
-import './BirdDetails.css'
+import "./BirdDetails.css";
+import binoculars from "../../binoculars.svg";
+import { Link } from "react-router-dom";
+import { useState } from 'react';
 
+const BirdDetails = ({ bird, addBird }) => {
+  const [newBird, setNewBird] = useState(bird);
 
-const BirdDetails = ({bird}) => {
   return (
-    <div className='bird-detail'>
+    <div className="bird-detail">
       <h3>{bird.comName}</h3>
       <p>{bird.sciName}</p>
       <p>Last seen on: {bird.obsDt}</p>
       <p>How many? {bird.howMany}</p>
-      <button>🔍</button>
+      <Link to="/myBirds">
+        <img
+          className="binoculars"
```

Comment:
These are more cute than the mag glass. I think we need to add "I see it!" so it's more claer

Reply:
Great idea, I'll add this before we merge.

Reply:
Thank you for doing this